### PR TITLE
use HHVM to call wrapper binary on HHVM

### PR DESCRIPTION
**Pull request by ldct**
- [x] CR @mflores13

### CODE REVIEW

File: src/ParaTest/Runners/PHPUnit/Worker.php [hunk 1]
```diff
@@ -32,7 +32,12 @@ public function start($wrapperBinary, $token = 1, $uniqueToken = null)
         if ($uniqueToken) {
             $bin .= "UNIQUE_TEST_TOKEN=$uniqueToken ";
         }
-        $bin .= "exec \"$wrapperBinary\"";
+        if (defined('HHVM_VERSION')) {
```
Comment (mflores13) [inline src/ParaTest/Runners/PHPUnit/Worker.php:35]: Is there another boolean way to see if we're on HHVM than to check `HHVM_VERSION`?

File: src/ParaTest/Runners/PHPUnit/Worker.php [hunk 2]
```diff
@@ -32,7 +32,12 @@ public function start($wrapperBinary, $token = 1, $uniqueToken = null)
...
+            $bin .= "LD_LIBRARY_PATH=/opt/pcrelib ";
+            $bin .= "exec hhvm -d max_execution_time=0 \"$wrapperBinary\"";
```
Comment (mflores13) [inline src/ParaTest/Runners/PHPUnit/Worker.php]: Would single quotes here prevent having to escape the inner double quotes?

File: src/ParaTest/Runners/PHPUnit/Worker.php (hunk 1 above)
Comment (ldct) [inline src/ParaTest/Runners/PHPUnit/Worker.php:35]: no, this is the recommended way http://hhvm.com/blog/2393/hhvm-2-3-0-and-travis-ci

**Comment by mflores13** [comment]
Small nits, but otherwise, :+1:

File: src/ParaTest/Runners/PHPUnit/Worker.php (hunk 2 above)
Comment (ldct) [inline src/ParaTest/Runners/PHPUnit/Worker.php]: fixed

**Pull request merged**